tests/test_transformers.py

```diff
@@ -89,3 +89,23 @@ def test_transformers_raise_notfitted_transform(transformer):
     X, y = load_moscow_stjoes(return_X_y=True)
     with pytest.raises(NotFittedError):
         transformer().transform(X)
+
+
+@pytest.mark.parametrize("transformer", [CCATransformer, CCorATransformer])
```

Comment:
Do you want to set a module level list like `TEST_TRANSFORMERS` or do you prefer this syntax?

Reply:
Sounds good to me.  I've gone with `TEST_ORDINATION_TRANSFORMERS` for now.  "ordination" was the best term that I could think of to differentiate these two transformers from the ones that retain all of their covariates.